Controller/TwigExceptionController.php

```diff
@@ -28,31 +27,13 @@ class TwigExceptionController extends TemplatingExceptionController
     protected function createView($format, FlattenException $exception, $code, $parameters, Request $request, $showException)
     {
         $view = parent::createView($format, $exception, $code, $parameters, $request, $showException);
-
-        if ($this->getViewHandler()->isFormatTemplating($format)) {
-            $view->setTemplate($this->findTemplate($request, $format, $code, $showException));
-            $view->setData($parameters);
-        }
+        $view->setTemplate($this->findTemplate($request, $format, $code, $showException));
```

Comment:
What do you think about searching the available templates in a compiler pass instead to avoid calls to the template engine which probably cost a lot of resources ?